src/main/resources/wordlisttest.txt

```diff
@@ -0,0 +1,4 @@
+cat
```

Comment:
Nice that you've created a more manageable dataset to test with - however can you think of another way to achieve that without having to create lots of wordlist files?